app/controllers/concerns/defendant_helpers.rb

```diff
@@ -19,7 +23,7 @@ def set_defendant_if_required
   end
 
   def term
-    @defendant_identifier || defendant_params[:id]
+    @defendant_asn_or_nino || defendant_params[:id]
```

Comment:
```suggestion
    defendant_asn_or_nino || defendant_params[:id]
```

Reply:
I cannot make this change, as it breaks linking. I think the issue is the defendant_controller and laa_references_controller work slightly differently. In the laa_references_controller I have had to introduce @defendant_asn_or_nino as params[:id] is sometimes null, for example after failed validation, hence `f.hidden_field(:defendant_asn_or_nino, value: (params[:id] || @defendant_asn_or_nino))` in the view. In laa_references_controller I need to use @defendant_asn_or_nino in load_and_authorize_search to set the defendant (in order to then get the id to send to the adaptor for linking etc.)

Reply:
what is value of id? is it `uuid` or `asn_or_nino`?

Reply:
@jsugarman id is asn_or_nino, defendant_id is uuid.